hydradx/notebooks/Money Market/test_hdx_price_drop.ipynb

```diff
@@ -0,0 +1,744 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e16957de-5d62-4ae4-b2cc-2fc9763e1537",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\jepid\\PycharmProjects\\HydraDX-simulations\\hydradx\\notebooks\\Money Market\n",
+      "C:\\Users\\jepid\\PycharmProjects\\HydraDX-simulations\\hydradx\\model\n"
+     ]
+    }
+   ],
+   "source": [
+    "import json\n",
+    "import os\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "print(os.getcwd())\n",
+    "while not os.path.exists('./model'):\n",
+    "    os.chdir('..')\n",
+    "os.chdir('model')\n",
+    "print(os.getcwd())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "id": "dbc70760-27be-4aa2-9220-9a38f20e6bca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdx_prices = {}\n",
+    "hdx_price_files = [f for f in os.listdir(\"./data/prices\") if f.lower().startswith(f\"hdx asset state \")]\n",
+    "for file_name in hdx_price_files:\n",
+    "    with open(f\"./data/prices/{file_name}\", 'r') as file:\n",
+    "        hdx_prices.update(json.load(file))\n",
+    "\n",
+    "# usd_prices = {}\n",
+    "# usd_price_files = [f for f in os.listdir(\"./data/prices\") if f.lower().startswith(f\"usdt asset state \")]\n",
+    "# for file_name in usd_price_files:\n",
+    "#     with open(f\"./data/prices/{file_name}\", 'r') as file:\n",
+    "#         usd_prices.update(json.load(file))"
+   ]
+  },
```

Comment:
When I try to run this I get the following error:
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
Input In [2], in <cell line: 2>()
      1 hdx_prices = {}
----> 2 hdx_price_files = [f for f in os.listdir("./data/prices") if f.lower().startswith(f"hdx asset state ")]
      3 for file_name in hdx_price_files:
      4     with open(f"./data/prices/{file_name}", 'r') as file:

FileNotFoundError: [Errno 2] No such file or directory: './data/prices'